examples/go/README.md

```diff
@@ -0,0 +1,77 @@
+# Writing an SSE plugin using Go
+
+General background information is found here:
+
+* [Writing an SSE Plugin](../../docs/writing_a_plugin.md)
+* [Protocol Description](../../docs/SSE_Protocol.md) (API reference)
+* Documentation and tutorials for [gRPC](http://www.grpc.io/docs/) and [protobuf](https://developers.google.com/protocol-buffers/docs/overview)
+
+## Getting started
+First of all, you need an installation of the [Go](https://golang.org/) programming language. 
+
+Next, you need to download and install the [protobuf compiler](https://github.com/google/protobuf/releases) and add the *protoc* executable to your %PATH%. Then, to install the protoc plugin for Go, execute:
+
+*go get -u github.com/golang/protobuf/protoc-gen-go*
+
+Note that the *protoc-gen-go* program also needs to be in your %PATH%.
+
+After setting up your [Go](https://golang.org/) environment, open a command prompt and execute: 
+
+*go get github.com/qlikmats/server-side-extension/examples/go/basic_example*
+
+You will get an "error" saying there are no go files in the \gen folder, but that's  fine - we will generate a file in that folder in the following step.
+
+From your Go SSE plugin folder (i.e. %GOPATH%\src\github.com\qlikmats\server-side-extension\examples\go\basic_example), execute:
+
+*go generate*
+
+This will generate the gRPC/Protobuf file(s) that your server will use  for implementing the qlik.sse.Connector service.
+
+
+
+### Configuring QlikSense to use the sample gRPC server
+By default, the Go sample plug-in runs on port 50051 on localhost, so for a QlikSense Desktop installation, the following should be added to settings.ini:
+
+[Settings 7] 
+
+SSEPlugin=SSE_Go, localhost:50051[,PATH_TO_CERTIFICATE_FOLDER]
+
+Note that the string SSE_Go is the identifier that will prefix all plug-in functions when they are called from within Qlik.
+Use a different identifier for your own plug-in, and remember that this exact string has to be used for your Sense applications to work with the extension.
+
+The address (localhost:50051) should of course match the address in the server's configuration file.
+
+For single-machine development and testing it is OK to use unsecure communication, but for production scenarios you should use certificates. See [Generating certificates](../../generate_certs_guide/README.md).
+
+## Starting the server
+
+The Go gRPC sample server can be built and started using the go command:
```

Comment:
More of a script than a go command.
Also, we should aim for platform independence and the script is windows only.

Reply:
I was thinking of having Windows/Linux scripts, but I can scrap the scripts altogether and just document the command line(s).